docs/content/reference/cli/glooctl.md

```diff
@@ -38,7 +38,7 @@ glooctl is the unified CLI for Gloo.
 * [glooctl completion](../glooctl_completion)	 - generate auto completion for your shell
 * [glooctl create](../glooctl_create)	 - Create a Gloo resource
 * [glooctl dashboard](../glooctl_dashboard)	 - Open Gloo dashboard
-* [glooctl debug](../glooctl_debug)	 - Debug a Gloo resource (requires Gloo running on Kubernetes)
+* [glooctl debug](../glooctl_debug)	 - Debug Gloo (requires Gloo running on Kubernetes)
```

Comment:
minor request - could we say Gloo Gateway instead of Gloo?

unless this is specifically meant to be product agnostic (AKA if i also have GME running in the same cluster & this command could be meant for that too)